crates/file-explorer/src/fs.rs

```diff
@@ -133,7 +136,7 @@ pub async fn open(path: PathBuf) -> Result<Entry> {
 
     Ok(Entry::File(Box::new(File::new(path, file, metadata))))
 }
-
+#[allow(unused_variables)]
```

Comment:
```suggestion
```

Reply:
I don't know why there is dead code in the crate file-explorer but the warnings are present.